ante/fee.go

```diff
@@ -5,26 +5,41 @@ import (
 
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	paramtypes "github.com/cosmos/cosmos-sdk/x/params/types"
 	tmstrings "github.com/tendermint/tendermint/libs/strings"
+
+	"github.com/cosmos/gaia/v8/x/globalfee"
+	"github.com/cosmos/gaia/v8/x/globalfee/types"
 )
 
 const maxBypassMinFeeMsgGasUsage = uint64(200_000)
 
+var defaultZeroGlobalFee = []sdk.DecCoin{sdk.NewDecCoinFromDec("uatom", sdk.NewDec(0))}
```

Comment:
This should change into a function that accepts `ctx` as a parameter so it can access the `BondDenom` of the staking module.

Something like this:

```go
func DefaultZeroGlobalFee(ctx sdk.Context) string {
	return []sdk.DecCoin{sdk.NewDecCoinFromDec(stakingKeeper.BondDenom(ctx), sdk.NewDec(0))} 
}
```